man/loader.conf.xml

```diff
@@ -196,6 +196,28 @@
         by using the <keycap>f</keycap> key.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term>bitlocker</term>
```

Comment:
Please give this a more descriptive name. I'm pretty sure that this commit will not be the last time we have to deal with bitlocker, so a single-word description (and a noun in particular), might give us a headache in the future. Maybe "reboot-for-bitlocker" or something like that.